frontend/app/component/form/data_entry/candidates_votes/CandidatesVotesForm.test.tsx

```diff
@@ -123,9 +117,9 @@ describe("Test CandidatesVotesForm", () => {
 
       renderForm({ recounted: false });
 
-      const candidate1 = await screen.findByRole("textbox", {
-        name: `1 ${candidateName(politicalGroupMockData.candidates[0] as Candidate)}`,
-      });
+      const candidateNames = getCandidateFullNamesFromMockData(politicalGroupMockData);
+
+      const candidate1 = await screen.findByRole("textbox", { name: `1 ${candidateNames[0]}` });
```

Comment:
Why not add the index prefix to the return value of `getCandidateFullNamesFromMockData`?

Reply:
Because that would result in a single function that goes from mock data to an aria-label that's the concatenation of two columns in the UI. That seems to me like too many things (and something too specific) for a single function.